stdlib/asm/sys.masm

```diff
@@ -0,0 +1,26 @@
+proc.stack_cleanup_proc.16
+    popw.local.0
+    popw.local.1
+    popw.local.2
+    popw.local.3
+    push.env.sdepth
+    push.16
+    u32gt.unsafe
```

Comment:
I think we can optimize this a bit. We know that stack depth will never drop below 16, so if stack depth is not 16, we know that it must be greater than 16. This means we should be able to replace "u32gt.unsafe" (which takes 6 VM cycles) with `neq` (which takes 2 VM cycles). We can actually replace `push.16 u32gt.unsafe` with just `neq.16`.

Reply:
Thank you!